CLR/Core/CLR_RT_HeapBlock.cpp

```diff
@@ -1897,21 +1897,6 @@ HRESULT CLR_RT_HeapBlock::TransparentProxyValidate() const
 #endif //TINYCLR_APPDOMAINS
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 
-#if defined(PLATFORM_WINDOWS) || defined(PLATFORM_WINCE)
-
-static void MyAssert()
-{
-    NATIVE_PROFILE_CLR_CORE();
-    DebugBreak();
-}
-
-#else
-
-static void MyAssert() {}
-
-#endif

```

Comment:
No, we shouldn't be doing that. It just makes for more noise and is often wrong when the logic of the condition changes. But it doesn't matter here as this is code being *DELTETED*